src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/PolymorphicNuMTFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.filtering;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.lang.mutable.MutableBoolean;
+import org.apache.commons.math3.distribution.PoissonDistribution;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.util.Collections;
+import java.util.List;
+
+public class PolymorphicNuMTFilter extends HardFilter {
+    private static final double LOWER_BOUND_PROB = .01;
+    private static final double MULTIPLE_COPIES_MULTIPLIER = 1.5;
+    private int maxAltDepthCutoff;
+
+    public PolymorphicNuMTFilter(final double medianAutosomalCoverage){
+        if (medianAutosomalCoverage != 0) {
+            final PoissonDistribution autosomalCoverage = new PoissonDistribution(medianAutosomalCoverage * MULTIPLE_COPIES_MULTIPLIER);
+            maxAltDepthCutoff = autosomalCoverage.inverseCumulativeProbability(1 - LOWER_BOUND_PROB);
+        } else {
+            maxAltDepthCutoff = 0;
+        }
+    }
+
+    @Override
+    public ErrorType errorType() { return ErrorType.ARTIFACT; }
```

Comment:
This should be `ErrorType.NON_SOMATIC` because it's a mapping error of real DNA.